bundles/org.openhab.binding.tado/src/main/java/org/openhab/binding/tado/internal/builder/TerminationConditionBuilder.java

```diff
@@ -47,19 +51,19 @@ public TerminationConditionBuilder withTerminationType(OverlayTerminationConditi
         if (terminationType != OverlayTerminationConditionType.TIMER) {
             timerDurationInSeconds = null;
         }
-
         return this;
     }
 
-    public TerminationConditionBuilder withTimerDurationInSeconds(Integer timerDurationInSeconds) {
+    public TerminationConditionBuilder withTimerDurationInSeconds(@Nullable Integer timerDurationInSeconds) {
```

Comment:
Like in previous comment, I would suggest separate method if there is a need to clear duration minutes, and avoid `@Nullable` here.